.project

```diff
@@ -1,3 +1,3 @@
 {
-	'srcDirectory' : 'packages'
+	'srcDirectory' : 'src'
```

Comment:
Is this ok?

Reply:
should stay at "packages", you could just change it in the file and commit to sort it out. you have to be extra sure that the git working copy in the file system is clean before doing any commits externally (at least that's how I think this came to happen)